scripts/jira-automation.py

```diff
@@ -597,6 +761,17 @@ def main():
     if dry_run:
         logging.info("DRY_RUN is enabled")
 
+    parser = argparse.ArgumentParser(description='Snyk to Jira automation script')
```

Comment:
I guess it can go to the main entrance of this python script, usually where you call the main function.

e.g.: https://github.com/spolti/rhoai-model-pipeline/blob/main/scripts/deploy-isvc.py#L64